fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -634,4 +637,8 @@ public void loop() {
             }
         }
     }
+
+    private boolean isChangeCatalog(String identifier) {
```

Comment:
I think you should implement this in one place.
1. to reduce duplicate code
2. to reduce unnecessary operation.